projects/canopy/src/lib/modal/modal.notes.ts

```diff
@@ -0,0 +1,89 @@
+export const notes = `
+# Modal Component
+
+
+## Purpose
+Provides a modal component for displaying content.
+
+## Usage
+~~~js
+@NgModule({
+  ...
+  imports: [ ..., LgModalModule ],
+})
+~~~
+
+and in your HTML:
+
+~~~html
+<button lgModalTrigger="modal-story" lg-button type="button" [variant]="'outline-primary'">Open modal</button>
+
+<lg-modal id="modal-story" [closeOnOverlayClick]="closeOnOverlayClick" >
```

Comment:
I think I mentioned previously but I wonder if we could get a design/uk call on...
 - closeOnOverlayClick
 - showCloseButton

 It could be flexibility that we need, but if we don't, I wonder if we can drop them for simplicity and consistency.

Reply:
I would definitely keep the `showCloseButton` because if you have a short modal that has a heading, one single line of text and the "close" and "cta" buttons the cross at the top is completely unnecessary and it actually makes it an annoyance for users using screen readers.

I've added `closeOnOverlayClick` because I remembered stakeholders asking for that exact functionality when I was working on a previous project. It's when a user has to make a decision.

@alexcanning do you have any views on both please?

Reply:
We've written some guidance around the close button which can be read here -https://landg.atlassian.net/wiki/spaces/CAN/pages/857617703/Modal+Dialogue. It basically says, always show the close button.

I'd also have thought that clicking outside of the modal would be expected behaviour - performing the same dismiss action as close.

Reply:
Perfect, I'll update that :)